sonaranalyzer-dotnet/its/expected/akka.net/Akka.Cluster-{6AB00F61-269A-4501-B06A-026707F000A7}-S1200.json

```diff
@@ -28,33 +28,7 @@
 },
 {
 "id":  "S1200",
-"message":  "Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 26 to the maximum authorized 20 or less.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka.Cluster\ClusterEvent.cs",
-"region":  {
-"startLine":  657,
-"startColumn":  18,
-"endLine":  657,
-"endColumn":  45
-}
-}
-},
-{
-"id":  "S1200",
-"message":  "Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 23 to the maximum authorized 20 or less.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka.Cluster\ClusterHeartbeat.cs",
-"region":  {
-"startLine":  39,
-"startColumn":  27,
-"endLine":  39,
-"endColumn":  49
-}
-}
-},
-{
-"id":  "S1200",
-"message":  "Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 30 to the maximum authorized 20 or less.",
+"message":  "Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 31 to the maximum authorized 30 or less.",
```

Comment:
The number of detected dependencies for this class went up (by one). Is this expected?

Reply:
It is normal, we started to count some things (method return types for example) and stopped counting other - bounded generic types (e.g. IDictionary<int> and IDictionary<string> are counted as +1, not +2 as before).